servers/src/main/java/tachyon/master/block/ContainerIdGenerator.java

```diff
@@ -13,9 +13,8 @@
  * the License.
  */
 
-package tachyon.master;
+package tachyon.master.block;
 
-/** Type of Image entry. */
-enum ImageElementType {
-  Version, Checkpoint, InodeFile, InodeFolder, RawTable, Dependency,
+public interface ContainerIdGenerator {
```

Comment:
I am not sure why we need this interface, and its required method is implemented by BlockIdGenereator